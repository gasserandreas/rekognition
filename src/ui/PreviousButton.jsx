import React from 'react';
import styled from 'styled-components';

import { Previous } from 'grommet-icons';

import { Colors } from '../styles';

export const StyledPrevious = styled(Previous)`
  display: block;
  visibility: visible;

  &:hover {
    stroke: ${Colors.ColorsPalette.TextFaded};
    cursor: pointer;
  }
`;

/** Render go previous button as left arrow icon */
const PreviousButton = (props) => <StyledPrevious {...props} />;

export default PreviousButton;
