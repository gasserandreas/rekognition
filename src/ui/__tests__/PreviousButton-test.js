/* global React, shallow, mount, render, renderer, testSnapshot */

import PreviousButton, { StyledPrevious } from '../PreviousButton';

// test styling
it('PreviousButton should render correctly', () => {
  testSnapshot(<PreviousButton />);
});

it('PreviousButton styles should be consistent', () => {
  testSnapshot(<StyledPrevious />);
});