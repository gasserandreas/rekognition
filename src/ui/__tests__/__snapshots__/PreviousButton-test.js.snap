// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviousButton should render correctly 1`] = `
<svg
  aria-label="Previous"
  className="sc-bdVaJa kidtcg StyledIcon-ofa7kd-0 iOkQrb"
  viewBox="0 0 24 24"
>
  <polyline
    fill="none"
    points="7 2 17 12 7 22"
    stroke="#000"
    strokeWidth="2"
    transform="matrix(-1 0 0 1 24 0)"
  />
</svg>
`;

exports[`PreviousButton styles should be consistent 1`] = `
<svg
  aria-label="Previous"
  className="sc-bdVaJa kidtcg StyledIcon-ofa7kd-0 iOkQrb"
  viewBox="0 0 24 24"
>
  <polyline
    fill="none"
    points="7 2 17 12 7 22"
    stroke="#000"
    strokeWidth="2"
    transform="matrix(-1 0 0 1 24 0)"
  />
</svg>
`;
